contents/tutorials/nps-survey.md

```diff
@@ -0,0 +1,102 @@
+---
+title: How to measure your NPS score in PostHog
+date: 2023-08-16
+author: ["joe-martin"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-4.png
+tags: ['surveys', 'insights']
+---
+
+One of the most well-known and useful types of survey for businesses is a Net Promoter Score ([NPS](https://en.wikipedia.org/wiki/Net_promoter_score_)) survey, which can act as a metric for user satisfaction and growth. In its simplest form it involves asking users how much they like your product or service, so you can gauge [product-market fit](/blog/measure-product-market-fit). 
+
+This may seem superficially straightforward, but it’s important to follow the methodology and phrasing precisely in order to collect a valid result. 
```

Comment:
```suggestion
This may seem straightforward, but it’s important to follow the methodology and phrasing precisely to collect a valid result. 
```
On first glance, I don't know what "superficially" means here.